test/fixtures/auth/keys.fixtures.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+
+const keys = {
+  valid: `-----BEGIN PUBLIC KEY-----
+MIICIjANBgkqhkiG9w0BAQEFAAOCAg8AMIICCgKCAgEAxHWGhSkfYnbgHYKJ/KVG
+dBJ3pMoRsbl43x/H+0knzBl0IKR7Btk2PR5VGk9xelOCQQmyepyG0nfycThm5+1U
+DFFv/WMRMg3YHKocHMJ78HvjBhzBETWnM3xywmHqmDPc8RKPLsjVwpLn0ohJ94V7
+qz3ghttTJN7OquAELkxOHxb6O9lJcCcku/ZbttggrP32abh5KhpCtl/5sodMixiA
+frx+s6NIdyvlYj8Y2UmhyPYzso/tCqiYByBZPHmjrJDTQHgppgU+gMjbrek8pl9u
+P555Al2ejqJ0hRtPfEW09hh3jcr4G8ltEisJLJsay4D2DrkbIT6XPi75o79203xz
+llod62g9Bcwi8jqdonRRyWIdswhFjX4CtscLxdllQaaPtorVGDuKJdByBJq6adTB
+wH8YKBMNEOflZAUFiRVnYPYMZ4IEhRCyWk6u6U+5OdQlg6vGHd2TcQ3xU1qbTu+I
+QyB8SePMRd5XkZfd0D/stAjtrH4mKre5fNBd7G9g8xhXov4UzXjrfm5hfcvJQAPG
+63EDWTd39zAIAee1DUT8ZXj58YaEKYcZEOm2q/z/JF+D0WG6YVktJlGrO7DSOFAA
+FC04z/LXuPbCi5MgNIx8c2ulfuHPXzysKTYmN0jv5vBYrU9VofSP35OeUjz24WBf
+JgiKJMLoQ8MMgLNHOPS3bQECAwEAAQ==
+-----END PUBLIC KEY-----`,
+  invalid: `-----BEGIN PUBLIC KEY-----
+MIICIjANBgkqhkiG9w0BAQEFAAOCAg8AMIICCgKCAgEAt6tCBlQXzoev09uUdWwL
+Sqo2ZMbA/SCnqH6IaFuxph3aWrFCJmoY5ns27N7iVyogzfY73lG+2gA4jNP7vvsh
+PwHArD3KcGOCd1W2NN/XaYLSVGWgOpS54imDwL36etJA4baexfp3eacoZHr8H+U3
+Z18nKxZ/EsoaqOrS3aWFjhDOwUiUFKivAUCWC4QtlLFn3arBekmmz0qas2lWwQBo
+nYBmtM49NsaNajFOUZLGQg+YYYoH1eXBiUk/DRCbfbM2IXx6Ox1jhMeS0pfImg+o
+eX8O1hbQv8EPCEpegb7mUz0XTu0lOa9kQ/q1dyZrljoGN1uQZE/WjYZvNWzyuyi7
+L0b3eSlT61tGL3H02YAOjC3fb/DpzwEnOWYaNx6YyeQ73peuQZFaNLYwYIf3RaEJ
+BzFuwovQijigkJxb9kPfzXTaCJ4oehPC86Q6EL7h4YkwtG+qzGdi2Nqbk68puXnT
+B7oKhngeT4uBgUpxwX2rQitYKNCrd6vep6b3qYLYpW8h7qBEEMaldgdYNYSPZjdN
+c5P4/odRQTznCK7AOlAKkQOPjHBTVq+xYggzTpTZ5ucIPl0I7j1BiG47fW31y53M
+AAFH/95wDSSsIjaANLgQ3lt4e6uQfyG3mQxm3Is6bXdOHm2p/1kIXPHpz/3HkTZF
+irw0QLM3sgCuFug+CM2AzbUCAwEAAQ==
+-----END PUBLIC KEY-----`,
+  severalKeys: `-----BEGIN PUBLIC KEY-----
```

Comment:
You should perhaps use a better name here like you do with the two others. Something link `severalValidAndInvalid` or better. It's verbose but easy to read and to understand that you will use the fallback feature.